packages/playground/website/vite.config.ts

```diff
@@ -134,6 +135,13 @@ export default defineConfig(({ command, mode }) => {
 					}
 				},
 			} as Plugin,
+			/**
+			 * Generate a list of files needed for the website to function offline.
+			 */
+			listAssetsRequiredForOfflineModePlugin({
```

Comment:
Nitpick: maybe let's call it just `listAssetsRequiredForOfflineMode`? The word `Plugin` makes it seem like there's an offline mode plugin somewhere.